docs/reference/ccr/getting-started.asciidoc

```diff
@@ -1,7 +1,320 @@
 [role="xpack"]
 [testenv="platinum"]
-[[ccr-getting-started]]
-== Getting Started
+[[ccr-gs]]
+== Getting Started with Cross-Cluster Replication
 
 beta[]
-This is the getting started section of the {ccr} docs.
\ No newline at end of file
+
+This getting-started guide for {ccr} shows you how to:
+
+* <<ccr-gs-remote-cluster,Connect a local cluster to a remote cluster>>
+* <<ccr-gs-leader-index,Create a leader index>> in a remote cluster that can be
+  replicated to a follower index in a local cluster
+* <<ccr-gs-follower-index,Create a follower index>> that replicates a leader
+  index in a remote cluster
+* <<ccr-gs-auto-follow,Automatically create follower indices>> that replicate
+  new indices in a remote cluster
+
+[float]
+[[ccr-gs-before-you-begin]]
+=== Before you begin
+. {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[Install {es}]
+  your on local and remote clusters.
+
+. Obtain a license that includes the {ccr} features. See
+  {stack-ov}/license-management.html[License Management].
+
+. If {security} is enabled in your local and remote cluster, you need a user
+  that has appropriate authority to perform the steps in this tutorial.
++
+--
+[[ccr-gs-security]]
+The {ccr} features use cluster privileges and built-in roles to make it easier
+to control which users have authority to manage {ccr}.
+
+By default, you can perform all of the steps in this getting-started guide by
+using the built-in `elastic` user. However, a password must be set for this user
+before the user can do anything. For information about how to set that password,
+see {stack-ov}/security-getting-started.html[Getting started with security].
+
+If you are performing these steps in a production environment, take extra care
+because the `elastic` user has the `superuser` role and you could inadvertently
+make significant changes. You can alternately assign the appropriate privileges
+to a user ID of your choice.
+
+On the remote cluster containing the leader index, a user will need the
+`read_ccr` cluster privilege, and `monitor` and `read` privileges on the leader
+index.
+
+[source,yml]
+--------------------------------------------------
+ccr_user:
+  cluster:
+    - read_ccr
+  indices:
+    - names: [ 'leader-index' ]
+      privileges:
+        - monitor
+        - read
+--------------------------------------------------
+
+On the local cluster containing the follower index, the same user will need the
+`manage_ccr` cluster privilege, and `monitor`, `read`, `write` and
+`manage_follow_index` privileges on the follower index.
+
+[source,yml]
+--------------------------------------------------
+ccr_user:
+  cluster:
+    - manage_ccr
+  indices:
+    - names: [ 'follower-index' ]
+      privileges:
+        - monitor
+        - read
+        - write
+        - manage_follow_index
+--------------------------------------------------
+--
```

Comment:
I think you'll also need manage cluster privileges to change the cluster settings.